app/views/viewandamend/ViewAmendSelectorView.scala.html

```diff
@@ -20,12 +20,22 @@
   layout: templates.Layout,
   formHelper: FormWithCSRF,
   govukButton: GovukButton,
-  govukRadios: GovukRadios
+  govukRadios: GovukRadios,
+  govukErrorSummary: GovukErrorSummary
 )
 
 @(qtReference: TransferId, pstr: models.PstrNumber, qtStatus: models.QtStatus, versionNumber: String)(implicit request: Request[_], messages: Messages)
 
 @layout(pageTitle = messages("viewAmend.title"), showCYAFooter = true) {
+  @request.flash.get("error").map { _ =>
```

Comment:
The established pattern for implementing across the service is
` @if(form.errors.nonEmpty) {
            @govukErrorSummary(ErrorSummaryViewModel(form))
        }`